src/app/(proper_react)/redesign/(authenticated)/admin/feature-flags/page.tsx

```diff
@@ -37,16 +40,50 @@ export default async function FeatureFlagPage() {
             <th>Dependencies</th>
             <th>Allow List</th>
             <th>Wait List</th>
+            <th>Owner</th>
           </tr>
         </thead>
         <tbody>
           {data.map((item) => (
             <tr key={item.name}>
               <td>{item.name}</td>
-              <td>{String(item.is_enabled)}</td>
-              <td>{item.dependencies}</td>
-              <td>{item.allow_list?.join(", ")}</td>
-              <td>{item.wait_list?.join(", ")}</td>
+              <td>
+                <ToggleFlagEnabled
+                  id="isEnabled"
+                  name={item.name}
```

Comment:
Q: Does this work? It seems from the code below that _everything_ will have a `name` of `item.name`:
- `id="isEnabled` (L52)
- `id="dependencies"` (L62)
- `id="allowList"` (L69)
- `id="waitList"` (L76)
- `id="owner"` (L83)

Reply:
`item.name` should be the name of the flag, I believe.

Reply:
Yes, but isn't it giving every single `<input>` the same `name=` value then (despite having different `id=` values)?
Maybe I'm wrong, it just looks a bit suspicious.

Reply:
The way it's intended to work is that each `<input>` element has an `id` to identify what it is, and each row of elements has a `name` indicating the name of the flag that it modifies.